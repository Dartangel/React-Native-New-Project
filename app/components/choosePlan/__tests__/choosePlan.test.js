import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import ChoosePlan from '../index';

describe('ChoosePlan', () => {
    test('ChoosePlan snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<ChoosePlan />);
        expect(snapshot).toMatchSnapshot();
    });
});