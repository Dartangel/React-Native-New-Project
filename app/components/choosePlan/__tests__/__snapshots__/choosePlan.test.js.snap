// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChoosePlan ChoosePlan snapshot 1`] = `
<View
  style={
    Object {
      "alignContent": "center",
      "alignItems": "center",
      "backgroundColor": "rgba(0, 72, 82, 0.6)",
      "borderRadius": 25,
      "height": "17%",
      "justifyContent": "center",
      "marginBottom": 20,
      "marginHorizontal": 20,
      "paddingVertical": 20,
    }
  }
>
  <TouchableOpacity
    activeOpacity={0.2}
  >
    <Text
      style={
        Object {
          "alignItems": "center",
          "color": "#ffffff",
          "fontSize": 20,
          "justifyContent": "center",
        }
      }
    />
    <Text
      style={
        Object {
          "alignItems": "center",
          "color": "#ffffff",
          "fontSize": 15,
          "justifyContent": "center",
        }
      }
    />
  </TouchableOpacity>
</View>
`;
